### PR TITLE
Changes to nuspec files to work with 4.8 .Net Framework

### PR DESCRIPTION
**Pull request by alfredo-pozo**
Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

This PR is to update the generated Nuget packages of Dynamo to 4.8 .Net Framework and is related to the JIRA task https://jira.autodesk.com/browse/DYN-2685

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@QilongTang 
@mjkkirschner

### CODE REVIEW

File: tools/NuGet/template-artifactory/DynamoVisualProgramming.DynamoCoreRuntime.nuspec [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
-    <id>DynamoVisualProgramming.DynamoCoreRuntime</id>
-    <version>$Version$</version>
-    <authors>Autodesk</authors>
-    <owners>Autodesk</owners>
-    <licenseUrl>https://www.apache.org/licenses/LICENSE-2.0</licenseUrl>
-    <projectUrl>https://github.com/DynamoDS/Dynamo</projectUrl>
-    <iconUrl>https://raw.githubusercontent.com/DynamoDS/Dynamo/master/doc/distrib/Images/logo_square_32x32.png</iconUrl>
-    <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>This package bundles all files from the Dynamo bin folder except for test libraries and symbol files.</description>
-    <copyright>Copyright Autodesk 2019</copyright>
```
Comment (mjkkirschner) [inline tools/NuGet/template-artifactory/DynamoVisualProgramming.DynamoCoreRuntime.nuspec:13]: I guess we can also update these to 2020?

File: tools/NuGet/template-nuget/DynamoVisualProgramming.Core.nuspec [hunk 2]
```diff
@@ -1,43 +1,46 @@
...
+            <group targetFramework=".NETFramework4.8">
+                <dependency id="Prism" version="4.1"/>
+                <dependency id="NUnit" version="2.6.3"/>
+                <dependency id="DynamoVisualProgramming.ZeroTouchLibrary" version="$Version$"/>
+                <dependency id="Newtonsoft.Json" version="8.0.3"/>
+            </group>
+        </dependencies>
+    </metadata>
+    <files>
+        <file src="DSIronPython.dll" target="lib\net48" />
```
Comment (QilongTang) [inline tools/NuGet/template-nuget/DynamoVisualProgramming.Core.nuspec:34]: @mjkkirschner Do you recall why we include IronPython in nuget? Do we need to include CPython now?

**Comment by mjkkirschner** [comment]
Yeah was wondering this as well - let’s not add more here and instead shrink the Api when we can

File: tools/NuGet/template-artifactory/DynamoVisualProgramming.DynamoCoreRuntime.nuspec (hunk 1 above)
Comment (alfredo-pozo) [reply tools/NuGet/template-artifactory/DynamoVisualProgramming.DynamoCoreRuntime.nuspec:13]: Yes, I'll do the change

Comment (alfredo-pozo) [reply tools/NuGet/template-artifactory/DynamoVisualProgramming.DynamoCoreRuntime.nuspec:13]: Change done

**Comment by QilongTang** [comment]
@alfarok @mjkkirschner Merging this

**Pull request merged**